packages/backend/src/graphql/schema.graphql

```diff
@@ -66,6 +66,20 @@ type Query {
     "Filter webhook events based on specific criteria."
     filter: WebhookEventFilter
   ): WebhookEventsConnection!
+
+  "Fetch a page of combined payments"
+  payments(
```

Comment:
I named everything `CombinedPayments` in the backend because it more accurately reflects how the data is modeled (it's a composite of two separate things). I used `payments` in the gql schema because I dont think the frontend really cares about that. Still, I'm considering if `combinedPayments` is good here too for explicitness.